utilities/logger/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export {Core} from './core';
+export {Server} from './server';
```

Comment:
+1, assumed it would be factored out into seperate module at later stage as we need three different implementations.
I ll drop client and server and implement core only for now